### PR TITLE
Add GHA for TM Integration Tests

### PR DESCRIPTION
**Pull request by shamrickus**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue. 

## Which Traffic Control components are affected by this PR?

- CI tests

## What is the best way to verify this PR?
Look at GHA, make a branch and push to test run.

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: .github/actions/to-integration-tests/entrypoint.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-+set -e
-+
+set -e
+
```
Comment (zrhoffman) [inline .github/actions/to-integration-tests/entrypoint.sh:20]: This is #5952. Can this fix go in a separate PR?

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: .github/actions/to-integration-tests/entrypoint.sh (hunk 1 above)
Comment (shamrickus) [reply .github/actions/to-integration-tests/entrypoint.sh:20]: yes

**Review by zrhoffman — CHANGES_REQUESTED** [review]
The *Run integration tests* step of the *TM_Integration_tests* GHA is filled with this warning at different timestamps:

```go
[...]
Traffic Monitor: WARNING: towrap.go:657: 2021-06-17T00:39:06.536715282Z: Creating monitor config: CRConfig monitor trafficmonitor missing IP6 field
Traffic Monitor: WARNING: towrap.go:657: 2021-06-17T00:39:06.565512899Z: Creating monitor config: CRConfig monitor trafficmonitor missing IP6 field
Traffic Monitor: WARNING: towrap.go:657: 2021-06-17T00:39:06.59481632Z: Creating monitor config: CRConfig monitor trafficmonitor missing IP6 field
Traffic Monitor: WARNING: towrap.go:657: 2021-06-17T00:39:06.623785938Z: Creating monitor config: CRConfig monitor trafficmonitor missing IP6 field
Traffic Monitor: WARNING: towrap.go:657: 2021-06-17T00:39:06.652891157Z: Creating monitor config: CRConfig monitor trafficmonitor missing IP6 field
[...]
```

Maybe from
https://github.com/apache/trafficcontrol/blob/a60c85419d5cdbb8bf869275e7b30507d1588d23/traffic_monitor/tests/_integration/snapshot.json#L39

**Comment by shamrickus** [comment]
Fixed the jq command so those warnings aren't output.

**Review by zrhoffman — APPROVED** [review]
Nice work! Now we'll get the TM integration tests' code coverage on TM PRs.

**Review by zrhoffman — COMMENTED** [review]
It [fail](https://github.com/apache/trafficcontrol/runs/2852510348)s on the master branch at 3d36d9b38a:

```
Able to reach testcaches:30000/_astats
Able to reach http://trafficmonitor:8009/api/version
INFO: traffic_monitor_test.go:50: 2021-06-17T19:51:52.422700503Z: Using Config values:
			   TM Config File:       traffic-monitor-test.conf
			   TM URL:               http://trafficmonitor:8009
			   TM Session Timeout:   30\n
=== RUN   TestClient
--- PASS: TestClient (0.02s)
=== RUN   TestKBPS
    kbps_test.go:73: monitor bandwidth kbps expected 5000-20000 actual 2577.741903
--- FAIL: TestKBPS (5.00s)
FAIL
exit status 1
FAIL	github.com/apache/trafficcontrol/traffic_monitor/tests/_integration	6.027s
Error: Process completed with exit code 1.
```

I wonder if bandwidth is runner-specific.
